app/api/controllers/video.py

```diff
@@ -0,0 +1,118 @@
+from flask import request
+from flask_restplus import Resource, Namespace
+import json
+
+from app.api.dao.section_dao import SectionDAO
+from app.database.models.category import CategoryModel
+from app.api.models.video import *
+from app.api.validations.video import validate_video_creation_data
+from app.api.dao.video_dao import VideoDAO
+from app.api.dao.author_dao import AuthorDAO
+from datetime import datetime
+from app.api.mappers.video_mapper import map_to_dto
+from app.api.middlewares.auth import token_required
+
+video_ns = Namespace("videos", description="Video Library")
+add_models_to_namespace(video_ns)
+
+
+@video_ns.route("/add_category")
```

Comment:
@NenadPantelic this has been moved, please reflect the latest changes

Reply:
Sure :+1: